connector_easypost/unit/import_synchronizer.py

```diff
@@ -314,21 +315,37 @@ def run(self, odoo_binding_id):
                        self.backend_record.id)
 
 
-def _get_env_return_importer(env, model_name, backend_id):
+class ObjectDict(object):
```

Comment:
Hmmmm I think maybe this needs its own file in `unit`. I feel like it's going to be lost here & it's not only related to the import (tests also use)